client/src/templates/Challenges/components/prism-formatted.tsx

```diff
@@ -32,7 +32,4 @@ class PrismFormatted extends Component {
   }
 }
 
-PrismFormatted.displayName = 'PrismFormatted';
```

Comment:
Does getting rid of the `displayName` negatively affect the dev experience? I have never used it in personal projects.

Reply:
I’d say so. This is very helpful in debugging. I think I deleted it by mistake. Good catch!